Prace_domowe/Praca_domowa1/Grupa2/Tomek_Nocoń/WUMPD1.ipynb

```diff
@@ -0,0 +1,935 @@
+{
```

Comment:
<p>Bardzo dobrze że dodałeś to do EDA. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>